src/dynarec/rv64/dynarec_rv64_66.c

```diff
@@ -125,7 +125,13 @@ uintptr_t dynarec64_66(dynarec_rv64_t* dyn, uintptr_t addr, uintptr_t ip, int ni
             break;
         case 0x0F:
             switch(rep) {
-                case 0: addr = dynarec64_660F(dyn, addr, ip, ninst, rex, ok, need_epilog); break;
+                case 0: {
+                    if (rv64_vector)
+                        addr = dynarec64_660F_vector(dyn, addr, ip, ninst, rex, ok, need_epilog);
```

Comment:
That means we need to duplicate ALL the opcode in _vector and regular... Why not having regular version as a fallback if it's implemented in the _vector one instead?

Reply:
mmm not sure what you mean, this PR does have the regular version as fallback.

Reply:
What happens if the program have 66 0F 5B opcode for example? for now, it will be run by the interpreter, not the dynarec.

Reply:
it will not, see the end of `dynarec64_660F_vector()`, called `dynarec64_660F` as fallback there. maybe you have a better way to do the fallback in mind?

Reply:
Oh, ok. but I don't think it's the best place, because there can be many "default" there....

Reply:
so you have a better place? maybe in here, and use some pointer variable to indicate if it needs to fallback?

Reply:
yes, here is a better place.
there is already a way to return if the opcode was not handled, with `ok`... But you can also return 0 instead of addr when opcode is not handled, and have here the return value stored in a temp value, if it's not 0, return temp, else, falback to regular version.

Reply:
Done.